CONTRIBUTING.md

```diff
@@ -9,4 +9,9 @@ Any contributions are expected to include unit tests. These can be validated wit
 
 The go version in the `go.mod` is the currently supported version of go.
 
+The project uses remote buf packages which will require a one-time registry configuration:
+```shell
+export GOPRIVATE=buf.build/gen/go
```

Comment:
Do I need to add this to use flagd now?

Reply:
> Do I need to add this to use flagd now?

To be completely clear, this is only need to _build_ not _use_ flagd.